src/main/java/org/broadinstitute/hellbender/tools/dataflow/MarkDuplicatesDataflow.java

```diff
@@ -218,20 +269,48 @@ public void processElement(final ProcessContext context) throws Exception {
                             }
 
                             //order by score
-                            final List<PairedEnds> scored = Ordering.natural().reverse().onResultOf((PairedEnds pair) -> pair.score()).immutableSortedCopy(paired.get(true));
+                            List<PairedEnds> scored = Ordering.natural().reverse().onResultOf((PairedEnds pair) -> pair.score()).sortedCopy(paired.get(true));
```

Comment:
It looks like it's not actually important to have the sorted scored list, just to select the best one.  Could this be replaced with:

Optional<PairedEnds> best = paired.get(true).stream().max(Comparator.comparing(PairedEnds::score));